worker/src/fetch.ts

```diff
@@ -187,7 +187,7 @@ export class FetchHandler {
     if (url.pathname.endsWith('/')) {
       let jsonapi = await this.getDirectoryListing(url);
       if (!jsonapi) {
-        new Response(
+        return new Response(
```

Comment:
@burieberry this was the actual bug you saw around creating a new card instance when the directory doesn't exist yet--we forgot to return this response, so the client logic that checks to see if the directory was even there was blowing up because the response was not json api